Extensions/ParticleSystem/ExtensionSubDeclaration1.cpp

```diff
@@ -267,4 +267,22 @@ void ExtensionSubDeclaration1(gd::ObjectMetadata& obj) {
                    "CppPlatform/Extensions/particleSystemicon16.png")
       .AddParameter("object", _("Object"), "ParticleEmitter")
       .UseStandardRelationalOperatorParameters("number");
+
+  obj.AddAction("StartEmission",
+                _("Start emission"),
+                _("Refull tank (if not infinite) and start emission of the particles."),
```

Comment:
```suggestion
                _("Refill tank (if not infinite) and start emission of the particles."),
```

Reply:
Oups, I forgot that I could accept this suggestion directly. Thanks!